src/twisted/internet/test/test_inlinecb.py

```diff
@@ -145,3 +145,34 @@ def calling():
         d = calling()
         f = self.failureResultOf(d)
         self.assertIn("in erroring", f.getTraceback())
+
+
+    def test_forwardLotsOfTracebacks(self):
+        """
+        Test that several chained inlineCallbacks gives information about all generators
```

Comment:
Docstring here is OK, but try to reword it so that `Test that` is not there, since it is obvious that it is a test.  Twisted prefers to follow
docstring conventions here: https://jml.io/pages/test-docstrings.html